asv.conf.json

```diff
@@ -1,157 +1,12 @@
 {
-    // The version of the config file format.  Do not change, unless
```

Comment:
Why strip down this config file? I'd leave this change out. Having a documented base config is kind of useful for this sort of thing.

Reply:
Ok